test/api-test.js

```diff
@@ -0,0 +1,68 @@
+const request = require('supertest');
+const express = require('express');
+const router = require('../api/routes.js');
+const app = express();
+
+const mary = "mary";
+const jack = "jack";
+const err = true;
+const errObj = new Error("Response does not contain the correct payload");
+
+app.use('/', router);
+const server = app.listen(5000);
```

Comment:
Pretty sure this line holds the process from existing when the tests ends.

Reply:
How will the tests work if the app doesn't listen for connections? Supertest depends on open connections in order to send http requests.